presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2006,6 +2012,14 @@ public FeaturesConfig setMaterializedViewDataConsistencyEnabled(boolean material
         return this;
     }
 
+    @Config("materialized-view-partition-filtering-enabled")
```

Comment:
The current config name is confusing. lets rename it to something else, how about?
"consider-query-filters-for-materialized-view-partitions"